web_programming/get_imdb_top_250_movies_csv.py

```diff
@@ -0,0 +1,29 @@
+import csv
+from typing import Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_imdb_top_250_movies(url: str = "") -> Dict[str, float]:
```

Comment:
Can we get some `doctests` here

Reply:
added doc strings

Reply:
This is network access code.  I am not clear how we would doctest it without resorting to mocking or really slowing down our test cycles, both of which might be overkill.  Note that most of the algorithms in the __ftp__ and __web_programming__ directories do not have doctests and I am OK with that.  Thoughts?

Reply:
See https://www.python.org/dev/peps/pep-0008/#documentation-strings for conventions on docstrings.  psi/black will change ''' to """ so you might as well do that.

Reply:
@cclauss I am noob, I have doubt whenever I want to upload a new file should open a new branch and make the pull request?

Reply:
```
git checkout master
git pull --rebase upstream master
git push origin master
git branch
git checkout -b my_new_idea
```

Reply:
thanks